### DISCUSSION TITLE
Limit number of messages to show

### DISCUSSION THREAD

**Discussion by izzap** [augustoproiete/serilog-sinks-richtextbox#52] [Q&A]
How do I limit the amount of data in the richtextbox? So i for instance only show the latest 1000 messages

**Comment by davidgyates** [comment]
For me, logger starts to run really slowly at 100,000 messages at >2GB memory usage and just grows from there - slower and slower.

If you're app is already in trouble, not being able to limit the number of logs will kill it completely.
